src/coreclr/tests/runtest.py

```diff
@@ -1002,7 +1002,7 @@ def setup_args(args):
         location using the build type and the arch.
     """
 
-    requires_coreroot = args.arch.lower() != "wasm"
+    requires_coreroot = args.arch.lower() != "wasm" and args.host_os.lower() != "android"
```

Comment:
This arbitrary mixing of "arch" vs. "host_os" checks is just asking for trouble - for instance, are we expecting to have "android wasm" tests at some point? I don't have any particular suggestions how to improve this right now, I'm mentioning this more or less as a heads-up that this might be worth clarifying at some point.

Reply:
Right now os="Browser" and arch="wasm" are logically equivalent, and will be that way for the foreseeable future. We could make both checks look at the host_os (just check for "Browser").

Reply:
Fixed, thanks!